Mathlib/Order/Filter/CardinalInter.lean

```diff
@@ -295,9 +296,9 @@ lemma cardinalInter_ofCardinalGenerate (hc : 2 < c) :
 
 variable {g}
 
-/-- A set is in the `cardinalInterFilter` generated by `g` if and only if
-it contains an intersection of `c` elements of `g`. -/
-theorem mem_cardinaleGenerate_iff {s : Set α} {hreg : c.IsRegular} :
+/-- If `c` is regular then a set is in the `CardinalInterFilter` generated by `g` if and only if it
```

Comment:
Same here about the doc-string change: please revert or split out

Reply:
Done